lib/mongo/collection/view/change_stream.rb

```diff
@@ -238,7 +229,30 @@ def closed?
         # @since 2.5.0
         def inspect
           "#<Mongo::Collection::View:ChangeStream:0x#{object_id} filters=#{@change_stream_filters} " +
-            "options=#{@options} resume_token=#{@resume_token}>"
+            "options=#{@options} resume_token=#{resume_token}>"
+        end
+
+        # Returns the resume token that the stream will
+        # use to automatically resume, if one exists.
+        #
+        # @example Get the change stream resume token.
+        #   stream.resume_token
+        #
+        # @return [ Hash | nil ] The change stream resume token.
+        # 
+        # @since 4.0.7
+        def resume_token
+          cursor_resume_token = @cursor.resume_token if @cursor
+          cursor_resume_token || @resume_token
+        end
+
+        # Executes a getMore command, for testing purposes.
+        #
+        # @api private
+        def get_more
```

Comment:
In Ruby we can invoke private methods directly:

<pre>
stream.instance_variable_get('@cursor').get_more
</pre>

Thus we don't need to add methods like this that are only used in the test suite.